File: Servers/HeadServer/HeadServer.cs
using System.Collections.Generic;
namespace HeadServer
{
    public class HeadServer
    {
        private string __dirname = "/usr/local/src/lamp";
        private List<string> gateways = new List<string>();
        private List<string> indexForSites = new List<string>();
        private string indexPageData;
        private List<string> oldGateways = new List<string>();
        private List<string> oldIndex = new List<string>();
        private int siteIndex;
        public HeadServer() {}

        private static void Main()
        {
            new HeadServer();
        }
    }
}

File: output/HeadServer.js
require('./mscorlib.debug.js');require('./CommonLibraries.js');require('./CommonServerLibraries.js');require('./Models.js');
require('./mscorlib.debug.js');require('./CommonLibraries.js');require('./CommonServerLibraries.js');require('./Models.js');
////////////////////////////////////////////////////////////////////////////////
// HeadServer.HeadServer
var $HeadServer_HeadServer = function() {
	this.$__dirname = '/usr/local/src/lamp';
	this.$gateways = [];
	this.$indexForSites = [];
	this.$indexPageData = null;
	this.$oldGateways = [];
	this.$oldIndex = [];
	this.$siteIndex = 0;
};
$HeadServer_HeadServer.$main = function() {
	new $HeadServer_HeadServer();
};
Type.registerClass(global, 'HeadServer.HeadServer', $HeadServer_HeadServer, Object);
$HeadServer_HeadServer.$main();
